app/models/manageiq/providers/azure/cloud_manager/cloud_database.rb

```diff
@@ -31,18 +33,25 @@ def self.params_for_create
           :id        => 'server',
           :name      => 'server',
           :label     => _('Database Server Name'),
+          :isRequired => true,
+          :validate   => [{:type => 'required'}],
         },
         {
-          :component => 'text-field',
-          :id        => 'resource_group',
-          :name      => 'resource_group',
-          :label     => _('Resource Group'),
+          :component  => 'text-field',
+          ## TODO: Conver this into a drop down, dynamically populate it
+          ## TODO: is this dependant on the provider or provider AND database type (selected above)
+          :id         => 'resource_group',
+          :name       => 'resource_group',
+          :label      => _('Resource Group'),
+          :isRequired => true,
+          :validate   => [{:type => 'required'}],
         },
       ],
     }
   end
 
   def self.raw_create_cloud_database(ext_management_system, options)
+    options.symbolize_keys!
```

Comment:
@MelsHyrule probably better to convert all of the `options[:key]` to `options["key"]` below rather than symbolizing keys here since we know the API will pass in string keys

Reply:
Ok, so this means that line 55, 68 and the `db_client.create` call on 68 would need to use string keys and hopefully they work with them.